bindings/nodejs/generated.js

```diff
@@ -36,7 +36,7 @@ function isMusl() {
   // For Node 10
   if (!process.report || typeof process.report.getReport !== 'function') {
     try {
-      const lddPath = require('child_process').execSync('which ldd').toString().trim();
+      const lddPath = require('child_process').execSync('which ldd').toString().trim()
```

Comment:
Please don't change this file by hand.

Reply:
I did not change it. Just ran `yarn build` and regenerated it, if I didn't commit this, the ci will occur an error.

Reply:
Got it.